server/data/migrations/dev/20190214184331_teachers_classes.js

```diff
@@ -0,0 +1,19 @@
+exports.up = function(knex, Promise) {
+	return knex.schema.createTable('teachers_classes', (tbl) => {
```

Comment:
I think we have this as classes_teachers in the other migrations? Should we change it? teachers_classes is more consistent with students_classes